x-pack/test/cases_api_integration/common/plugins/security_solution/server/plugin.ts

```diff
@@ -39,7 +39,12 @@ export class FixturePlugin implements Plugin<void, void, FixtureSetupDeps, Fixtu
       cases: ['securitySolutionFixture'],
       privileges: {
         all: {
-          api: ['casesSuggestUserProfiles', 'bulkGetUserProfiles'],
+          api: [
```

Comment:
We need to do the same for `x-pack/test/cases_api_integration/common/plugins/observability/server/plugin.ts`